internal/store/migrate.go

```diff
@@ -31,7 +31,7 @@ func (sqlStore *SQLStore) Migrate() error {
 	applied := 0
 	for _, migration := range migrations {
 		if !currentVersion.EQ(migration.fromVersion) {
-			break
+			continue
```

Comment:
Fix an issue with migrations not actually doing more than the first one ;P

Reply:
Haha, good catch!